admin/category.ts

```diff
@@ -18,15 +18,15 @@ export async function createCategory() {
 
   const program = await getProgram();
 
-  const categoryPk = await findCategoryPda(code, program);
+  const subcategoryPk = await findCategoryPda(code, program);
 
   const createCategoryArgs = {
     code: code,
     name: name,
   } as CreateCategoryArgs;
 
   const createCategoryAccounts = {
-    category: categoryPk,
+    category: subcategoryPk,
```

Comment:
should you rename `category` as well?

Reply:
ah the admin scripts need a bit of love, I will update them in my next PR